salt/iiif/config/usr-local-bin-loris-cache-clean-soft

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
+# adapted from: https://github.com/loris-imageserver/loris/blob/development/bin/loris-http_cache_clean.sh
```

Comment:
not sure if this is actually used anymore. Might remove it later if not

Reply:
I don't know much about the maintenance tasks. I switched them to use the syslog for logging which pruned away a chunk of complexity from the formula in general and helped with debugging. `journalctl` has a bunch of drawbacks but it is awfully convenient to just do `journalctl -f` and watch for errors during highstate.

Reply:
> not sure if this is actually used anymore

it looks like the cron job uses a script that wraps the `soft` and `hard` clean scripts and switches between them depending on what the current usage is. I won't touch them for this work but happy to take a second look after it's all done.